taipy/gui/viselements.json

```diff
@@ -1353,6 +1353,22 @@
                         "default_value": "False",
                         "doc": "If set to True, the progress value is shown."
                     },
+                    {
+                        "name": "title",
+                        "type": "dynamic(str)",
+                        "doc": "The title of the progress indicator."
+                    },
+                    {
+                        "name": "color",
+                        "type": "str",
+                        "doc": "The color of the progress indicator."
+                    },
+                    {
```

Comment:
Please move 'title_anchor' next (just after) 'title': they are so related.
My prefered order would be title, title_anchor, color.

Reply:
updated